features/delayed_job.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Bugsnag detects errors in Delayed job workers
+
+Background:
+  Given I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
+  And I set environment variable "MAZE_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
+  And I configure the bugsnag endpoint
+  And I generate the gem and put it in "delayed_job"
```

Comment:
Ben has done this in env.rb and I think I prefer that approach. Perhaps we should look at doing the same here.